src/ert/libres_facade.py

```diff
@@ -31,7 +31,10 @@
 
 
 class LibresFacade:  # pylint: disable=too-many-public-methods
-    """Facade for libres inside ERT."""
+    """Facade for libres inside ERT. The intention of this class is to expose
```

Comment:
I think `Facade for libres inside ERT` can be removed, I am sure future developers will be able to infer by reading the class name.

Reply:
guess that's a fair point. chucked it!